request_without_range.go

```diff
@@ -0,0 +1,56 @@
+package pget
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+
+	"github.com/cheggaaa/pb/v3"
+)
+
+func DownloadFiles(urls []string, dest string) error {
+	for _, link := range urls {
+		if err := DownloadFile(link, dest, context.Background()); err != nil {
```

Comment:
Please context move to the first parameter
https://pkg.go.dev/context#:~:text=The%20Context%20should%20be%20the%20first%20parameter%2C%20typically%20named%20ctx